doc/authoring_help.md

```diff
@@ -40,14 +40,10 @@ helps['account clear'] = """
                   long-summary: |
                       Longer summary with newlines preserved.  Preserving newlines is helpful for paragraph breaks.
                   populator-commands: 
-                  - az account list
-                  - These indicate where values can be retrieved for input to this command
```

Comment:
Why are we getting rid of this description? 
I had no idea `populator-commands` were a thing for our help until this pr. 
May be useful to keep in case more people want to use it.

Reply:
The example is fine (`az account list`) but the way it was written, with "help text" in the list, means that the CLI would actually display that as `Values from: These indicate where values can...`. 

A better way (for a future PR) would be to list and describe the kwargs that the help system supports.  This is what the command authoring documentation does.